packages/notification-service/src/notification-packages/previewUserGenerationCompleted.ts

```diff
@@ -2,6 +2,7 @@ import { KAFKA_TOPICS } from "@amplication/schema-registry";
 import { NotificationContext } from "../util/novuTypes";
 
 const NOTIFICATION_KEY = "complete-preview-onboarding";
+const SENDGRID_TEMPLATE_ID = "d-f8d34551e2c7436681b34f8af0782788";
```

Comment:
Should be this configure as environment variable? Is sendgrid supporting versioning?

Reply:
I don't think so, as it is specific to the notification package. 
Meaning, each notification package may have a different template ID, just like the NOTIFICATION_KEY

Reply:
I meant for testing in other envs. But it ok :)